src/utils.js

```diff
@@ -0,0 +1,15 @@
+findHighest = (a,b,c) => {
+    let highest = 0;
+    if(a > b) {
+      highest = a;
+    }else {
+      highest = b;  
+     }
+    if(c > highest)
+      highest = c;
+    
+    return highest;    
+};
+exports.findHighest = findHighest;
+
+'\n'
```

Comment:
I do not mean `'\n'` when I said insert a new line. I mean have an empty line at the end of the file, like this

```
const a = 5;

```
instead of 
```
const a = 5;
```

Notice the empty line at the end

Reply:
ok